management/mysql_test.go

```diff
@@ -79,6 +79,10 @@ func TestAddMySQL(t *testing.T) {
 	})
 
 	t.Run("All fields", func(t *testing.T) {
+
```

Comment:
Remove those empty lines at the start of functions. https://github.com/mvdan/gofumpt can do that automatically.